webapp-client/src/component/ContentPane.js

```diff
@@ -0,0 +1,78 @@
+import '../stylesheets/ContentPane.css';
+import errorify from '../errorify.js';
+import Item from './Item';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+export default class ContentPane extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      items: [],
+    };
+    errorify(this);
+  }
+
+  componentDidMount() {
+    this.setItems(this.props);
+  }
+
+  componentDidUpdate(prevProps) {
+    if (this.props !== prevProps) {
+      this.setItems(this.props);
+    }
+  }
+
+  async setItems(props) {
+    const { categoryId, defaultCategory, userId } = props;
+    let url;
+    let response;
+    let items;
```

Comment:
you can declare them on the same line

Reply:
I originally had it that way but eslint linted it as an error. I guess it's slightly more readable this way?